src/modules/Dropdown/Dropdown.js

```diff
@@ -751,7 +751,7 @@ export default class Dropdown extends Component {
   renderText = () => {
     const { multiple, placeholder, search, text } = this.props
     const { searchQuery, value, open } = this.state
-    const hasValue = multiple ? !_.isEmpty(value) : !!value
+    const hasValue = multiple ? !_.isEmpty(value) : value !== ''
```

Comment:
Actually I had `_.isNil(value)` my first version but then two related tests failed so i changed it to only check for empty strings. Maybe it should be:
```js
const hasValue = multiple 
  ? !_.isEmpty(value) 
  : !_.isNil(value) && value !== ''
```
I'll  update this PR as soon as possible 😉